src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseSource.iOS.cs

```diff
@@ -696,6 +701,11 @@ public NativeListViewBase Owner
 			}
 		}
 
+		/// <summary>
+		/// Determines if the cell is currently displayed in the collection
```

Comment:
the property doesnt have any side effect.
```suggestion
		/// Indicates if the cell is currently displayed in the collection.
```